tests/edgeless/frame/frame.spec.ts

```diff
@@ -234,7 +234,9 @@ test.describe('add element to frame and then move frame', () => {
       await assertEdgelessElementBound(page, frameId, [100, 100, 500, 500]);
     });
 
-    test('group should not be moved since its center is not in frame', async ({
+    // FIXME(@doouding): the group should not be created since some shape is in frame some is not
+    // we should add a `groupId` field for each element instead manage the group relationship in a standalone class
+    test.skip('group should not be moved since its center is not in frame', async ({
```

Comment:
I think this test can be removed, since in the new `group` concept, only the same level elements can be grouped